java/pgv-java-stub/src/test/java/io/envoyproxy/pgv/Assertions.java

```diff
@@ -0,0 +1,15 @@
+package io.envoyproxy.pgv;
+
+import org.assertj.core.api.ThrowableAssert.ThrowingCallable;
+
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+public final class Assertions {
+    private Assertions() {
+        // Intentionally left blank.
+    }
+
+    public static void assertValidationException(final ThrowingCallable f) {
+        assertThatThrownBy(f).isInstanceOf(ValidationException.class);
```

Comment:
Why is it necessary to wrap this line of code in its own class? If you are only going to use this method in `StringValidationTest`, you can move it to that class.

Reply:
I plan to update other test classes too but in a dedicated PR. I don't want to mix things and make it hard to review.